src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-type.mdx

```diff
@@ -783,6 +783,21 @@ These users can also see [logs in context](/docs/logs/logs-context/configure-log
 
 </CollapserGroup>
 
+## User type and roles [#user-type-and-roles]
+
+Here's an explanation of how user type and [role-based access](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts) differ: 
+
+* A user's **[user type](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type)**: What user type to make a user is a question of what an organization expects their team members to do with New Relic, and how much value they expect to get out of that work. It's mainly a billing-related decision. It sets the maximum allowed permissions a user can access. User type is **not** meant to be used for controlling a user's access and permissions: for that, you should use roles. 
```

Comment:
@zuluecho9 these bullets overlap a bit with the bullets in this doc, but a little overlap may drive home the point: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts.mdx